editor/views/editoritem.py

```diff
@@ -511,10 +511,51 @@ def get(self, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         ei = self.get_object()
+        error_message = ''
+        count = 1
```

Comment:
Python has a really good function called `enumerate` for situtations like this, where you want to count your way through a list of items.

Instead of

```
count = 1
for item in items:
    print("Item "+count+" is "+item)
    count += 1
```

you can do

```
for count, item in enumerate(items, 1):
    print("Item "+count+" is "+item)
```